src/libs/ReportUtils.ts

```diff
@@ -1980,27 +1789,26 @@ function getChatRoomSubtitle(report) {
     }
     if (getChatType(report) === CONST.REPORT.CHAT_TYPE.DOMAIN_ALL) {
         // The domainAll rooms are just #domainName, so we ignore the prefix '#' to get the domainName
-        return report.reportName.substring(1);
+        return report?.reportName?.substring(1) ?? '';
     }
-    if ((isPolicyExpenseChat(report) && report.isOwnPolicyExpenseChat) || isExpenseReport(report)) {
+    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
```

Comment:
Is this eslint-disable-next-line needed?

Reply:
It's needed because linter is complaining we are not using `??` operator even though we can't use it in `if` statement and we just want to use `||` operator